### PR TITLE
Pipe tools 3 compat

### PR DESCRIPTION
**Pull request by bitsofbits**
* Make compatible with pipe-tools-3 and Python 3
    - test with anchorage naming
         - [x] check that runs successfully on Python 3
         - [x] check that results match Python 2
    - test with port events
         - [x] check that runs successfully on Python 3
         - [x] check that results match Python 2
    - test with port visits
         - [x] check that runs successfully on Python 3
         - [x] check that results match Python 2
* Update anchorage tables
    - update anchorage overrides
    - add Peru list
    - don't filter out inshore anchorages (per @natemiller's request).
    -  don't replace Chinese anchorage names with S2ID when in the overrides list.

**Do Not Merge Till Fully Tested**

### CODE REVIEW

**Comment by bitsofbits** [comment]
Check that name anchorages returns same results under Python 3 as Python 2.

First check that the generated tables are the same size using 
```
select count(*) from machine_learning_dev_ttl_120d.named_anchorages_v20191205_py3
```
```>>> 166508 , 166508```

Then check that the values are all the same using
```
select a.*, b.label as b_label, b.sublabel as b_sublabel 
from machine_learning_dev_ttl_120d.named_anchorages_v20191205_py3 a
join machine_learning_dev_ttl_120d.named_anchorages_v20191205 b
using (s2id)
where a.label != b.label or a.sublabel != b.sublabel
```

**Comment by bitsofbits** [comment]
Checked that port events will run (but not checking results) with ```
    docker-compose run port_events \
        --job_name porteventstest \
        --input_table pipe_production_v20190502.position_messages_ \
        --anchorage_table anchorages.named_anchorages_v20191205 \
        --start_date 2017-12-01 \
        --end_date 2017-12-01 \
        --output_table machine_learning_dev_ttl_120d.new_pipeline_port_events_test_v20191209_py3 \
        --project world-fishing-827 \
        --max_num_workers 200 \
        --requirements_file requirements.txt \
        --project world-fishing-827 \
        --staging_location gs://machine-learning-dev-ttl-30d/anchorages/portevents/output/staging \
        --temp_location gs://machine-learning-dev-ttl-30d/anchorages/temp \
        --setup_file ./setup.py \
        --runner DataflowRunner \
        --disk_size_gb 100 \
        --start_padding 1
```

**Comment by bitsofbits** [comment]
Checked that port visits run (but didn't check results) with ```
    docker-compose run port_visits \
        --job_name portvisitssharded \
        --events_table machine_learning_dev_ttl_30d.new_pipeline_port_events_test_v20191209_py3 \
        --start_date 2017-12-01 \
        --end_date 2017-12-01 \
        --start_padding 365 \
        --output_table machine_learning_dev_ttl_30d.new_pipeline_port_visits_test_v20191209_py3 \
        --project world-fishing-827 \
        --max_num_workers 200 \
        --requirements_file requirements.txt \
        --project world-fishing-827 \
        --staging_location gs://machine-learning-dev-ttl-30d/anchorages/portevents/output/staging \
        --temp_location gs://machine-learning-dev-ttl-30d/anchorages/temp \
        --setup_file ./setup.py \
        --runner DataflowRunner \
        --disk_size_gb 100 
```

**Comment by bitsofbits** [comment]
Code for comparing tables:
```
with 

t1 as (
select * except (events) from `machine_learning_dev_ttl_120d.new_pipeline_port_visits_test_py27_v20191210*`
),

t2 as (
select * except (events) from `machine_learning_dev_ttl_120d.new_pipeline_port_visits_test_py37_v20191210*`
),

missing as (
(select "T1" as which, * from ((select * from t1) EXCEPT DISTINCT (select * from t2)))
UNION ALL
(select "T2" as which,  * from ((select * from t2) EXCEPT DISTINCT (select * from t1)))
)

-- select
--   (select count(*) from t1), (select count(*) from t2),
--   (select count(*) from missing)
  
select * from missing
-- order by timestamp
```

File: test/test_name_anchorages.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 import yaml
 import pytest
+import six
```
Comment (smpiano) [inline test/test_name_anchorages.py]: @bitsofbits Could be possible to delete the import of six if there is not used?

File: setup.py [hunk 2]
```diff
@@ -15,59 +16,12 @@
     'pyyaml',
     'unidecode',
     'numpy',
-    "pipe-tools==2.0.0",
+    "pipe-tools==3.0.0",
```
[the remark this answers is not in the document]
Comment (bitsofbits) [reply setup.py]: Good idea. I'll do that tomorrow.

File: pipe_anchorages/objects/namedtuples.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import json as ujson
```
[the remark this answers is not in the document]
Comment (bitsofbits) [reply pipe_anchorages/objects/namedtuples.py]: I think that's a good idea, but I'm inclined to wait till after this round of changes. I think the versions are slightly different and we may need to do some reconciliation and I'd rather do that as a separate PR.

File: test/test_name_anchorages.py (hunk 1 above)
Comment (bitsofbits) [reply test/test_name_anchorages.py]: Yep. Done!

**Pull request merged**